handsontable/src/helpers/dom/element.js

```diff
@@ -1148,3 +1148,30 @@ export function runWithSelectedContendEditableElement(element, callback, invisib
 
   removeContentEditableFromElementAndDeselect(element, invisibleSelection);
 }
+
+/**
+ * Creates a new DOM element and appends it to the parent element with the provided class name(s) and attributes.
+ *
+ * @param {HTMLElement} parentElement The parent element.
+ * @param {object} properties The properties object.
+ * @param {string} properties.tagName The type of the new element.
+ * @param {string} properties.className Class name as string or array of strings.
```

Comment:
```suggestion
 * @param {string|string[]} properties.className Class name as string or array of strings.
```

Reply:
Done in https://github.com/handsontable/handsontable/pull/10540/commits/ec631144f18cba5a3168886919502a0e5c80debf.